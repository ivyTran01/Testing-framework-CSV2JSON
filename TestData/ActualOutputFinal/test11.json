[{"field1":"Ivy","field2":22,"field3":true},{"field1":"Tanzil","field2":25,"field3":""},{"field1":"Danny","field2":21,"field3":true}]