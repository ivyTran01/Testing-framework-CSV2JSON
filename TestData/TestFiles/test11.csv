Ivy	22	true
Tanzil	25	""
Danny	21	true
