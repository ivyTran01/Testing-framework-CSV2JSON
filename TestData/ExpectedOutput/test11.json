[
 {
  "name":"Ivy  T.",
  "age":22,
  "field3":null,
  "height":5.2
 },
 {
  "name":"Tanzil",
  "age":25,
  "field3":false,
  "height":5.9
 },
 {
  "name":"Danny  Yu",
  "age":21,
  "field3":true,
  "height":5.11
 },
 {
  "name":"Abdullah  Amjad",
  "age":23,
  "field3":true,
  "height":5.1
 },
 {
  "name":"Jocelyn",
  "age":22,
  "field3":true,
  "height":5.2
 },
 {
  "name":"Christ",
  "age":23,
  "field3":false,
  "height":5.1
 }
]